docker-compose.yml

```diff
@@ -36,7 +36,6 @@ services:
   postgres:
     container_name: hedera-postgres
     image: postgres:10.9-alpine
-    stop_grace_period: 2m
```

Comment:
How would this help? This will bring down the wait to shutdown after SIGKILL is sent to 10 secs from 2 min right

Reply:
Locally, I keep having an intermittent problem with Docker not being able to kill the `hedera-postgres` container. 😕  Removing this longer grace period seems to reduce the frequency. 

Have you ever seen the `hedera-postgres` container refuse to stop?

Reply:
Not really.